packages/plugins/users-permissions/server/services/providers.js

```diff
@@ -43,6 +43,8 @@ module.exports = ({ strapi }) => {
   const connect = async (provider, query) => {
     const access_token = query.access_token || query.code || query.oauth_token;
 
+    console.dir({ query });
```

Comment:
@alexandrebodin I think you forgot to remove this 😝

Reply:
haha yes it's still a work in progress :D